content/en/blog/_posts/2025-01-22-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: blog
+title: "Introducing Kube-scheduler-simulator"
+date: 2030-01-22
```

Comment:
pick a reasonable date. I cannot see 2030 in the preview:

<img width="680" alt="image" src="https://github.com/user-attachments/assets/7e046c7d-09f8-4ce5-ae1e-b16d4ab68c09" />


Or tell me how to look at the preview